x-pack/plugins/uptime/kibana.json

```diff
@@ -23,12 +15,10 @@
   "server": true,
   "ui": true,
   "version": "8.0.0",
-  "requiredBundles": [
-    "observability",
-    "kibanaReact",
-    "kibanaUtils",
-    "home",
-    "data",
-    "ml"
-  ]
+  "requiredBundles": ["observability", "kibanaReact", "kibanaUtils", "home", "data", "ml"],
+  "owner": {
+    "name": "Uptime",
+    "githubTeam": "uptime"
+  },
+  "description": "This plugin visualizes data from Uptime and Elastic Synthetics, and integrates with other Observability solutions."
```

Comment:
I'd just say `from Synthetics and Heartbeat` since data doesn't come from `Uptime`